app/screens/InfoScreen/OurSponsorsScreen.tsx

```diff
@@ -11,18 +11,18 @@ import { Screen, Text } from "../../components"
 import { translate } from "../../i18n"
 import { useAppNavigation, useHeader } from "../../hooks"
 import { useSponsors } from "../../services/api"
-import { RawSponsor } from "../../services/api/webflow-api.types"
```

Comment:
Ended up renaming these types to generated types from: https://github.com/infinitered/ChainReactApp2023/pull/122

Here is the convention for collections now:
- `SponsorsCollection`: actual API response from Webflow
- `Sponsor`: data return type of `useSponsors`
- `CleanedSponsors`: return type of `cleanedSponsors`